exp/web-dashboard.md

```diff
@@ -0,0 +1,10 @@
+The Anbox Cloud dashboard offers a web interface to create, manage and stream applications. The dashboard is especially helpful for developers new to Anbox Cloud and helps them by offering a simpler and intuitive management interface.
+
+The dashboard may seem more oriented towards developers than end users. However, it only uses available APIs and SDKs and can be a good example to build your own web-based platform.
```

Comment:
Reading this sentence and the previous one, I'm not sure if I really understand what you want to say:
"The dashboard is especially helpful for developers... The dashboard may seem more oriented towards developers than end users"

Should it be the other way around?
"The dashboard may seem more oriented towards end users than developers"

Or am I misunderstanding what you mean to say?

Reply:
This was an existing sentence - It seems to be suggesting that the dashboard is more helpful for developers than end users because developers use it to launch applications and sessions. I think end-users here mean the customers of our customers. For example, if we have a customer selling a cloud gaming platform, it is useful for developers who create that platform than the end users who will be consuming the game on the platform.

Does that make sense? We can change it/drop it if it is too confusing.

Reply:
On second thoughts, I have dropped the statement. I think the concept is more clear without the statement.